src/views/FSL/lookup/lookup.vue

```diff
@@ -318,36 +319,50 @@
                       :percent="percentage(order)"
                     />
                   </div>
-                
-                  <div
-                    class="
-                    col-12 col-xs-2 col-md col-lg
-                    d-flex
-                    align-items-center
-                    justify-content-center
-                  "
-                  >
-                    <button
-                      :disabled="order.business_type === 'Cash n Carry'"
-                      :class="
-                        order.status === 'Completed' ? 'approved' : 'pending'
-                      "
-                      @click="displayAmortization(order)"
-                      class="btn status my-sm-2"
-                    >
-                      View Plan
-                      <i
-                        :class="
-                          order.status === 'Completed'
-                            ? 'fa-check-circle'
-                            : 'fa-hourglass-half'
-                        "
-                        class="fas ml-3"
-                        style="font-size: 1.4rem"
-                      ></i>
-                      <!--                                        // TODO:: cleanup-->
-                    </button>
-                  </div>
+                  
+                  <!-- <div class="
+    col-12 col-xs-2 col-md col-lg
+    d-flex
+    align-items-center
+    justify-content-center"
+    :key="index"
+>
+    <select
+        class="custom-select w-100"
+        data-vv-as="order_status"
+        data-vv-validate-on="blur"
+        :name="'order_status_' + index"
+        v-model="NewOrderAmortization.orderStatus"
+        v-validate="'required'"
+    >
+        <option selected disable value="Select Status">Select Status</option>
+        <option value="close">Close</option>
+        <option value="repossessed">Repossessed</option>
+    </select>
+</div> -->
+
+<div
+    class="
+        col-12 col-xs-2 col-md col-lg
+        d-flex
+        align-items-center
+        justify-content-center"
+>
+    <button
+        :disabled="order.business_type === 'Cash n Carry'"
+        :class="statusOrder(order.status)"
+        @click="displayAmortization(order)"
+        class="btn status my-sm-2"
+    >
+        View Plann
+        <i
```

Comment:
You need to take care of this too with the change in status

Reply:
I know that closed should have a check icon, which icon should name should repossessed have?

Reply:
Try any nice icon at fontawesome